src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -339,8 +356,33 @@
        :on-done (fn [{:keys [success? get-parsed-response]}]
                   (if success?
                     (do (swap! state assoc :is-leo-whitelisted? true)
-                        (this :-get-clusters-list-if-whitelisted))
+                        (this :-get-clusters-list-if-whitelisted)
+                        (this :-schedule-cookie-refresh))
                     (swap! state assoc :server-response {:server-error (get-parsed-response false)})))}))
+
+   :-schedule-cookie-refresh
+   (fn [{:keys [props state locals this]}]
+     (let [{:keys [server-response]} @state
+           {:keys [clusters]} server-response]
```

Comment:
For things like this, you can do a nested destructure:

```clojure
(let [{{:keys [clusters]} :server-response} @state]
  ...)
```

https://clojure.org/guides/destructuring

The jury is still out on whether or not this is too arcane for us.